internal/pkg/agent/application/upgrade/artifact/download/composed/verifier.go

```diff
@@ -31,13 +31,13 @@ func NewVerifier(verifiers ...download.Verifier) *Verifier {
 }
 
 // Verify checks the package from configured source.
-func (e *Verifier) Verify(a artifact.Artifact, version string) error {
+func (e *Verifier) Verify(a artifact.Artifact, version string, skipVerifyOverride bool, pgpBytes ...string) error {
```

Comment:
I suppose the `skipVerifyOverride` parameter has this form because of the `skip-verify` flag but I would find it less confusing if it was formulated as a "positive" (like verify) or even better:
-  we could just skip the verification from the caller 
- if the skip-verify flag is set we use a no-op verifier 
(whichever is easier)

Reply:
good point